packages/avatar/stories/avatar.stories.tsx

```diff
@@ -8,31 +9,28 @@ import {
   iconSizeXl
 } from "../../design-tokens/build/js/designTokens";
 import { serviceImg } from "./helpers/serviceImg";
+import { AvatarProps } from "../components/Avatar";
 
 const iconSizes = [iconSizeXs, iconSizeS, iconSizeM, iconSizeL, iconSizeXl];
 
 export default {
   title: "Graphic Elements/Avatar",
-  component: Avatar
-};
+  component: Avatar,
+  argTypes: {
+    size: {
+      options: ["xs", "s", "m", "l", "xl"],
+      control: {
+        type: "select"
+      }
+    }
+  }
+} as Meta;
 
-export const Default = () => <Avatar src={serviceImg} label="Kubernetes" />;
-
-export const AllSizes = () => (
-  <div
-    style={{
-      alignItems: "center",
-      display: "grid",
-      gridTemplateColumns: `${iconSizes.join(" ")}`,
-      gridGap: "1rem"
-    }}
-  >
-    <Avatar src={serviceImg} label="Kubernetes" size="xs" />
-    <Avatar src={serviceImg} label="Kubernetes" size="s" />
-    <Avatar src={serviceImg} label="Kubernetes" size="m" />
-    <Avatar src={serviceImg} label="Kubernetes" size="l" />
-    <Avatar src={serviceImg} label="Kubernetes" size="xl" />
-  </div>
+const Template: Story<AvatarProps> = args => (
+  <Avatar src={serviceImg} label="Kubernetes" {...args} />
 );
 
-export const EmptySrc = () => <Avatar src="" label="Kubernetes" size="xl" />;
+export const Default = Template.bind({});
+
+export const EmptySrc = Template.bind({});
```

Comment:
I don't know that I see the usefulness of this story as `src` is a required field. I think we could remove this entirely. Thoughts?

Reply:
I agree with you. Will remove.